shared/utility/strutil/banner.hpp

```diff
@@ -1,559 +0,0 @@
-                                          R"(      )",     R"(       )",
-                                          R"(        )",   R"(       )",
-                                          R"(       )",    R"(        )",
-                                          R"(       )",    R"(       )",
-                                          R"(   )",        R"(   )",
-                                          R"(     )",      R"(        )",
-                                          R"(     )",      R"(      )",
-                                          R"(         )",  R"(          )",
-                                          R"(       )",    R"(        )",
-                                          R"(        )",   R"(        )",
-                                          R"(        )",   R"(        )",
-                                          R"(        )",   R"(       )",
-                                          R"(        )",   R"(      )",
-                                          R"(        )",   R"(        )",
-                                          R"(       )",    R"(        )",
-                                          R"(        )",   R"(        )",
-                                          R"(        )",   R"(        )",
-                                          R"(         )",  R"(        )",
-                                          R"(          )", R"(              )",
-                                          R"(       )",    R"(         )",
-                                          R"(       )",    R"(     )",
-                                          R"(       )",    R"(     )",
-                                          R"(    )",       R"(|______|)",
-                                          R"(   )",        R"(       )",
-                                          R"(       )",    R"(      )",
-                                          R"(       )",    R"(      )",
-                                          R"(     )",      R"( |___/ )",
-                                          R"(       )",    R"(   )",
-                                          R"(|__/ )",      R"(      )",
-                                          R"(   )",        R"(           )",
-                                          R"(       )",    R"(       )",
-                                          R"(|_|    )",    R"(    |_|)",
-                                          R"(      )",     R"(     )",
-                                          R"(     )",      R"(       )",
-                                          R"(       )",    R"(          )",
-                                          R"(      )",     R"( |___/ )",
-                                          R"(     )",      R"(     )",
-                                          R"(|_|)",        R"(     )",
-                                          R"(      )",     R"()"}};
-
-    std::string banner(const std::string& input) {
```

Comment:
This file is very much used!
Where do you think the text at the start of literally every role comes from?

Reply:
I thought that, but everything compiles without it, and the term `banner.hpp` doesn't appear in the codebase. :shrug:

Reply:
@TrentHouliston We don't use `banner.hpp` for banners, we use `nuclear/roles/banner/bigtext.py`.

Reply:
:O